app/services/Cards.rb

```diff
@@ -0,0 +1,74 @@
+class Cards
+  include ActiveModel::Validations
+
+  YAKU = ["ハイカード","ワンペア","ツーペア","スリー・オブ・ア・カインド","ストレート","フラッシュ","フルハウス","フォー・オブ・ア・カインド","ストレートフラッシュ"]
+
+  attr_accessor :card , :hand ,:numbers , :suits ,:tramp ,:error
+
+  validates :card, format: /\A[HDSC]([1-9]|[1][0-3])\s[HDSC]([1-9]|[1][0-3])\s[HDSC]([1-9]|[1][0-3])\s[HDSC]([1-9]|[1][0-3])\s[HDSC]([1-9]|[1][0-3])\Z/
```

Comment:
This is not model, so you don't need to use model validation
Actually, model validation is not so easy to customize